wordpress-mobile/WordPress-iOS
# Jetpack conn screen fix

Fixes #4929 

This is a stopgap fix for the Jetpack connection screen on iPads. I separated out the refactoring of this legacy code into a separate issue: #6389 This PR addresses:

* The skewing of the Jetpack logo
* Previously offscreen elements having an alpha of zero when the soft keyboard is lowered
* Ensures the login form does not move down past the lower application window frame when rotating the device with the screen up.

To test on an iPad

1. Launch WPiOS and login to a self-hosted site (be sure to be logged out of WP.com)
2. Tap the "stats" menu item on the left
3. Verify the Jetpack logo is correct
4. Tap either the username or password text field and bring up the soft keyboard
5. Rotate the device to landscape or portrait (multiple times)
6. Dismiss the soft keyboard and verify the login form is still centered appropriately for the device orientation AND make sure all interface elements are visible if they were moved off screen
7. Rinse and repeat on multiple iPads/Sims (Pro, Air, Mini)

Would you mind giving this a quick review @kurzee?
